### ISSUE TITLE
rekinect causes a crash when joining P2P sandbox server

### ISSUE THREAD

**Issue by katsvg** [WilliamVenner/gmcl_rekinect#4]
creating a multiplayer game with rekinect (x64) injected works just fine and allows me to use the v2 kinect with ragdolls etc. 
a singleplayer game works aswell; when joining a p2p sandbox multiplayer game it just closes as soon the game sends client info

i've got both the v2.0 sdk and everything copied over in the right place, cant figure out if this is a serverside issue but i believe it'd just work anyway

**Comment by WilliamVenner** [comment]
@owovin That is a different issue I think, which was fixed in the newest version.

**Comment by Toasty-Engineer** [comment]
having the same issue, here's my panic log:

PanicInfo {
    payload: Any { .. },
    message: Some(
        Failed to get CLuaInterface,
    ),
    location: Location {
        file: "gmcl_rekinect\\src\\exbones.rs",
        line: 36,
        col: 21,
    },
    can_unwind: true,
    force_no_backtrace: false,
}

**Comment by Foxache** [comment]
Having the same issue, any fix?

**Comment by WilliamVenner** [comment]
I cannot reproduce the issue on Windows x64. I tried singleplayer and peer to peer, no problem. Perhaps you need to update rekinect.

**Comment by WilliamVenner** [comment]
Please can you give [v0.1.4](https://github.com/WilliamVenner/gmcl_rekinect/releases/tag/v0.1.4) a try?

**Comment by Foxache** [comment]
> Please can you give [v0.1.4](https://github.com/WilliamVenner/gmcl_rekinect/releases/tag/v0.1.4) a try?

Of course, I am busy for somewhere around 15 to 1h then i will test for you.

**Comment by Foxache** [comment]
> Please can you give [v0.1.4](https://github.com/WilliamVenner/gmcl_rekinect/releases/tag/v0.1.4) a try?

![Image](https://github.com/user-attachments/assets/74a37fe7-0d31-4813-abe4-f7572707bcec)
Rekinect doesnt see Gmod opening.. odd

EDIT: Changing to 32 bit branch
EDIT 2 ELECTRIC BOOGALOO: 32 bit allows ReKinect to... do its thing

**Comment by Foxache** [comment]
> Please can you give [v0.1.4](https://github.com/WilliamVenner/gmcl_rekinect/releases/tag/v0.1.4) a try?

32 bit branch , Latest ReKinect , 360 Sensor . 

![Image](https://github.com/user-attachments/assets/6901591f-eefb-4a3f-abb6-2825fbe5f516)

NVM scratch that! It brings up the sensor but no movement happens...

EDIT: works singleplayer , not P2P or Server hosted 
EDIT: now it doesnt work on singleplayer , whaaat? (on restart it works)
EDIT: It seems to not want to work after visiting any multiplayer instance

**Comment by Foxache** [comment]
Is this repository still active?

**Comment by Mur4ik2w0** [comment]
> > Please can you give [v0.1.4](https://github.com/WilliamVenner/gmcl_rekinect/releases/tag/v0.1.4) a try?
> 
> "Image" Rekinect doesnt see Gmod opening.. odd
> 
> EDIT: Changing to 32 bit branch EDIT 2 ELECTRIC BOOGALOO: 32 bit allows ReKinect to... do its thing
but it have to allow us to play with kinect on x64 ver of game, isn't it?
